client/jetpack-connect/signup.js

```diff
@@ -85,10 +85,16 @@ export class JetpackSignup extends Component {
 		} );
 	}
 
+	isWoo( props = this.props ) {
+		const { from } = props.authQuery;
```

Comment:
Should we alias `from` here and everywhere else where it occurs, as it's a reserved keyword?